drivers/tty/serdev/pibridge_trace.h

```diff
@@ -103,16 +155,37 @@ DECLARE_EVENT_CLASS(pibridge_send_crc_class,
 	)
 );
 
+/*
+ * pibridge_send_io_crc
+ *
+ * Info: the CRC of an IO packet to be sent.
+ * Time: before the CRC of an IO packet is sent on the wire.
+ */
 DEFINE_EVENT(pibridge_send_crc_class, pibridge_send_io_crc,
 	TP_PROTO(u8 crc),
 	TP_ARGS(crc)
 );
 
+/*
+ * pibridge_send_gate_crc
+ *
+ * Info: the CRC to send for the gateway packet.
+ * Time: before the CRC of a gateway packet is sent on the wire.
+ */
 DEFINE_EVENT(pibridge_send_crc_class, pibridge_send_gate_crc,
 	TP_PROTO(u8 crc),
 	TP_ARGS(crc)
 );
 
+/*
+ * pibridge_receive_crc_class
+ *
+ * Print the CRC that is part of a received packet as well as the CRC that has
+ * been calculated for this packet.
+ *
+ * crc: CRC received with the packet
+ * exp: CRC calculated for the received packet by receiver.
```

Comment:
s/exp/exp_crc/

Reply:
Never mind, I didn't realize that you're specifying the data emitted by the tracepoint, not its arguments.